tests/testthat/comparison_data/emissions/L251.ctrl.delete.csv

```diff
@@ -0,0 +1,5516 @@
+INPUT_TABLE
+Variable ID
+DelEmCtrl
+
```

Comment:
These header lines are also cryptic, and unlike in `L201.nonghg_steepness.csv` they're not prefixed with `#`.  Are these supposed to be comments, or do they mean something to the data system and get processed automatically?  If the latter, then do the comment characters need to be removed from `L201...` above?